packages/templates/src/test/raw.js

```diff
@@ -23,22 +23,19 @@ import { resolve, basename } from 'path';
 /**
  * Internal dependencies
  */
-import { PAGE_LAYOUT_TYPES } from '../../../edit-story/components/library/panes/pageLayouts/constants';
+import { PAGE_LAYOUT_TYPES } from '../../../../assets/src/edit-story/components/library/panes/pageLayouts/constants';
```

Comment:
Hmm we need to avoid this dependency on another (soon-to-be) package. Otherwise this will end up causing a circular dependency.

We could probably simply remove the second assertion below (the one using `PAGE_LAYOUT_TYPES`). Should be acceptable risk.

Reply:
To confirm, you want to remove this test? Maybe we could just copy and paste the value of this const over, for the test?

Reply:
Bit of a hacky solution but it works. It is just a test after all. https://github.com/google/web-stories-wp/pull/6252/commits/bb10452ec384853de9a0ebeddf9cf7a0e5b887bd

Reply:
This won't work forever. Now we have to maintain a list of page layout types in two places. That's not ideal.

My suggestion was not to remove the test, but to remove the second `expect(...)` assertion in it.

That's better than this manual list.

It was dynamic for good reasons.

Reply:
Removed in https://github.com/google/web-stories-wp/pull/6252/commits/be7959947a36ae6a0eb6f2e267bd5c694abbb65e